examples/chip-tool/commands/common/Command.cpp

```diff
@@ -395,3 +410,30 @@ void Command::WaitForResponse(uint16_t duration)
         ChipLogError(chipTool, "No response from device");
     }
 }
+
+#else // CONFIG_USE_SEPARATE_EVENTLOOP
+
+static void OnResponseTimeout(chip::System::Layer *, void *, chip::System::Error)
+{
+    ChipLogError(chipTool, "No response from device");
+
+    chip::DeviceLayer::PlatformMgr().StopEventLoopTask();
+}
+
+CHIP_ERROR Command::ScheduleWaitForResponse(uint16_t duration)
```

Comment:
Please rename or comment `duration` to indicate units.

Reply:
Renamed to `uint16_t seconds` for both wait functions.